README.md

```diff
@@ -1 +1,51 @@
-# registry-zookeeper
\ No newline at end of file
+# registry-zookeeper
+
+Some application runtime use Zookeeper as service discovery.
```

Comment:
Some application runtime use Zookeeper as service discovery. => Some applications use Zookeeper for service discovery.

Reply:
done